Sources/TuistKit/Mappers/Factories/WorkspaceMapperFactory.swift

```diff
@@ -41,14 +41,6 @@ final class WorkspaceMapperFactory: WorkspaceMapperFactorying {
         var mappers: [WorkspaceMapping] = []
         mappers.append(AutomationPathWorkspaceMapper(workspaceDirectory: workspaceDirectory))
         mappers += self.default(config: config)
-        if config.generationOptions.contains(.autogenerationOptions(.disabled)) {
-            mappers.append(
-                AutogeneratedWorkspaceSchemeWorkspaceMapper(
-                    codeCoverageMode: config.codeCoverageMode,
-                    testingOptions: []
-                )
-            )
-        }
```

Comment:
@fortmarek what was this for? Why do we need it for automation even if the scheme is disabled from config?

Reply:
this makes `tuist test` faster since we're able to run only one scheme with all the test targets instead of triggering them individually.

Reply:
I should have restored the functionality in [this comment](https://github.com/tuist/tuist/pull/4092/commits/6a128c504a23d4301dbf3055cc4f3e1a2c303e3b), makes sense?

Reply:
Looks good!